sdk/greengrass/event-stream-rpc-client/src/main/java/software/amazon/awssdk/eventstreamrpc/StreamResponse.java

```diff
@@ -10,20 +10,20 @@
     /**
      * Completable future indicating flush of the request that initiated the stream operation
      *
-     * @return {@link CompletableFuture}
```

Comment:
It is weird that all these docs changed. Why did this happen?

Reply:
Manual additions that will get stomped unfortunately.